gitlab/pkg/apis/sources/v1alpha1/gitlabsource_types.go

```diff
@@ -171,6 +182,26 @@ func (s *GitLabSourceStatus) MarkSecret() {
 	gitLabSourceCondSet.Manage(s).MarkTrue(GitLabSourceConditionSecretProvided)
 }
 
+// MarkWebhook sets the condition that the source has set its webhook configured.
+func (s *GitLabSourceStatus) MarkWebhook() {
+	gitLabSourceCondSet.Manage(s).MarkTrue(GitLabSourceConditionWebhookConfigured)
+}
+
+// MarkNoWebhook sets the condition that the source does not have its webhook configured.
+func (s *GitLabSourceStatus) MarkNoWebhook(reason, messageFormat string, messageA ...interface{}) {
+	gitLabSourceCondSet.Manage(s).MarkFalse(GitLabSourceConditionWebhookConfigured, reason, messageFormat, messageA...)
+}
+
+// MarkReceiveAdapter sets the condition that the source has its Receive Adapter service ready.
+func (s *GitLabSourceStatus) MarkReceiveAdapter() {
+	gitLabSourceCondSet.Manage(s).MarkTrue(GitLabSourceConditionReceiveAdapterConfigured)
+}
+
+// MarkNoReceiveAdapter sets the condition that the source doesn't have its Receive Adapter service ready.
+func (s *GitLabSourceStatus) MarkNoReceiveAdapter(reason, messageFormat string, messageA ...interface{}) {
+	gitLabSourceCondSet.Manage(s).MarkFalse(GitLabSourceConditionReceiveAdapterConfigured, reason, messageFormat, messageA...)
+}
+
```

Comment:
See previous comment regarding keeping the Godocs simple. It's only a suggestion.